src/app/asset-page/licenses.ts

```diff
@@ -3,36 +3,33 @@
  */
 
 export const licenses = [
-    { name: 'Attribution',
+    { name: 'Creative Commons: Attribution',
```

Comment:
The difference between the new and old values are mostly just the `Creative Commons:` bit at the beginning. Once exception is the `Public Domain Mark` at the bottom.